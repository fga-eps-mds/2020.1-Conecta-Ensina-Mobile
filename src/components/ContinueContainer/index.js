import React from 'react';
import CustomText from '../CustomText';
import Theme from '../../../Theme';
import {ButtonContainer, Button} from './styles';

export default function ContinueContainer({testID, onPress, marginTop}) {
  return (
    <Theme>
      <ButtonContainer marginTop={marginTop}>
        <Button testID={testID} onPress={onPress}>
          <CustomText white bigSmall>
            Continuar
          </CustomText>
        </Button>
      </ButtonContainer>
    </Theme>
  );
}
