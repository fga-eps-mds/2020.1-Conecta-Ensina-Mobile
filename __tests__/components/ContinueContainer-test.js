import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import ContinueContainer from '../../src/components/ContinueContainer';

it('renders correctly', () => {
  const tree = renderer.create(<ContinueContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
