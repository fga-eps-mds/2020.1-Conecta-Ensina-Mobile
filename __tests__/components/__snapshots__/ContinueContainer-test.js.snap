// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginTop": 0,
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "backgroundColor": "#297FB8",
        "borderRadius": 8,
        "height": 31,
        "justifyContent": "center",
        "opacity": 1,
        "width": 138,
      }
    }
  >
    <Text
      bigSmall={true}
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Roboto",
            "fontSize": 16,
            "textAlign": "center",
          },
        ]
      }
      white={true}
    >
      Continuar
    </Text>
  </View>
</View>
`;
